java/src/main/java/tech/sourced/enry/Enry.java

```diff
@@ -5,6 +5,8 @@
 import static tech.sourced.enry.GoUtils.*;
 
 public class Enry {
+    public static final Guess unknownLanguage = new Guess("UNK", false);
```

Comment:
We use empty string for unknown language in Go, not sure if it'd be fit well here. If so, I'd go with the same convention (could be `null` in Java)